### ISSUE TITLE
can't input ( because everything disappears

### ISSUE THREAD

**Issue by unek** [adonaac/thranduil#29]
until i add $. some pattern problem?

**Comment by adonaac** [comment]
Yup, it's some pattern problem. [, ], (, ) and @ break it. Already fixed it on another project I'm using but haven't merged it here yet. Will do eventually

**Comment by kyle-emmerich** [comment]
pls adonaac
